chandra_aca/maude_decom.py

```diff
@@ -601,6 +601,29 @@ def _group_packets(packets, discard=True):
         yield res
 
 
+def filter_vcdu_jumps(vcdu_counters):
+    """
+    Return a boolean mas to filter VCDU counters that are not continuous.
```

Comment:
I added a fix for this and included a corresponding test case.

The VCDU frame rolls over at 2^24. I used the maximum number of missing frames as 3600*4 as you said here, but why is this the right number?

Reply:
For the VCDU rollover, I see there a new test with what I would call "synthetic" VCDUs... from this work can you point me to a handy real rollover or should I go find one?

Reply:
The 3600 * 4 is a bit arbitrary. That corresponds to a one-hour comm loss. In the end it doesn't matter all that much since the worst case scenario (which is incredibly unlikely) is dropping one image.

Reply:
@jeanconn
```
ska_tdb.msids.find("vcdu", "counter")
dat = fetch.Msid("CVCDUCTR", "2023:001", "2023:120", stat="5min")
%matplotlib
dat.plot()
```
This is a bit interesting because it shows some resets in the safe modes.
<img width="418" alt="image" src="https://github.com/sot/chandra_aca/assets/348089/ddd559c9-6f09-4ab6-a47e-dc8044046fc4">

Reply:
@jeanconn there is a rollover between 2023:296:22:20 and 2023:296:22:22.

Reply:
So as an integration test I tried aca_view
```
python -m aca_view.scripts.aca_viewer --start 2023:296:22:20:00.000 --stop 2023:296:22:24:00.000 --maude
```
with this PR in PYTHONPATH, but I'm assuming that aca_view needs some fixes of its own to handle rollover, as it looks like aca_view won't play back after 2023:296:22:20:35.

Reply:
Let me check. Give me a few minutes.

Reply:
Thanks!  I don't know how far we need to get to the solution in this PR, but it looks like your changes helped the aca_view test a bit, in that it looks like the aca_view playback was no longer stuck at 2023:296:22:20:35 .  But in playback it also didn't show any new ACA data changes for the rest of the interval (time and att updating but all the ACA data stuck).

Reply:
This is not the behavior I see. I ran this:
```
python -m aca_view.scripts.aca_viewer --start 2023:296:22:20:26 --stop 2023:296:22:20:50
```
and I see the data updating after the rollover

Reply:
For me, I was running
```
python -m aca_view.scripts.aca_viewer --start 2023:296:22:20:00.000 --stop 2023:296:22:24:00.000 --maude
```
And it looks like it stops working at about 2023:296:22:20:40.301

Reply:
ok, I can confirm the issue in aca_view. One can check it by starting aca_view, opening the ipython widget, and typing:
```
from cxotime import CxoTime

CxoTime(timeline.table['TIME'][-1]).date
Out[3]: '2023:296:22:21:59.837'

CxoTime(timeline.slots[0].table['TIME'][-1]).date
Out[4]: '2023:296:22:20:39.887'
```

I think this is an aca_view issue, because running `chandra_aca.maude_decom.get_aca_images` returns correct data:
```
In [1]: from chandra_aca import maude_decom
   ...: from cxotime import CxoTime
   ...: start = '2023:296:22:20:26'
   ...: stop = '2023:296:22:22:00'
   ...: images = maude_decom.get_aca_images(start, stop)
   ...: print(CxoTime(images[images['IMGNUM'] == 0]['TIME'][-1]).date)
   ...: np.asarray(images[images['IMGNUM'] == 0]['IMG'][-1])

2023:296:22:21:55.914
Out[1]: 
array([[  31.  ,  112.  ,   91.75,   71.5 ,  139.  ,   98.5 ,   44.5 ,   17.5 ],
       [ 132.25,   44.5 ,  105.25,  267.25,  442.75,  226.75,   64.75,   31.  ],
       [  71.5 ,  105.25,  267.25,  706.  ,  753.25,  469.75,  166.  ,  132.25],
       [  91.75,  321.25, 1286.5 , 5788.75, 5289.25, 1192.  ,  280.75,  226.75],
       [  71.5 ,  280.75, 1246.  , 6855.25, 6841.75, 1711.75,  361.75,   98.5 ],
       [  24.25,   78.25,  287.5 , 1192.  , 1151.5 ,  510.25,  247.  ,   64.75],
       [  17.5 ,   64.75,  152.5 ,  152.5 ,  247.  ,  233.5 ,  145.75,   71.5 ],
       [  24.25,   24.25,   51.25,   71.5 ,  118.75,   37.75,   64.75,   44.5 ]])
```

Reply:
so I think the integration issue is not part of this PR.